.github/workflows/auto-comment.yaml

```diff
@@ -0,0 +1,22 @@
+name: Auto Comment
+on: [pull_request]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wow-actions/auto-comment@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          pullRequestOpened: |
+            👋 @{{ author }}
+            Once you merge this PR, please make sure you have follow these steps for CLI to pick up your changes.
+            1. Go to [SSH CI build]({{ https://dev.azure.com/devdiv/OnlineServices/_build?definitionId=17611&_a=summary }}).
+            2. Rerun the build by targeting "main" branch with "PublishNpmPackage" and "PublishNugetPackage" check boxes checked.
+            3. Once build is completed, the build number is your new ssh package number.
+            4. Go to SDK repo and send a new Pr with following changes made:
+              4.1 Update the "DevTunnelSshPackageVersion" [here in SDK repo]({{ https://github.com/microsoft/dev-tunnels/blob/main/cs/build/build.props }})
+              4.2 Update all "microsoft/dev-tunnels-ssh" and "microsoft/dev-tunnels-ssh-tcp" depenencies in the sdk repo with new version number.
```

Comment:
After 4.2, should we run `npm install`  to update the `ts/package-lock.json` file in the SDK repo?

Reply:
Added as step 4.3